src/program.ts

```diff
@@ -1306,6 +1344,29 @@ export class Program extends DiagnosticEmitter {
     }
   }
 
+  private addExportToImportNameSpace(
+    importNamespacesMap: Map<File, Map<File, Array<string>>>,
```

Comment:
Looks like we can avoid one lookup here by directly providing `Map<File, Array<string>>` as the argument, because earlier code already looked that up and knows that `file` is the key to the outer map iiuc. Depends a bit on the previous comment though, i.e. perhaps we can eliminate this helper with another approach to linking alias namespaces?

Reply:
Ah, yep, Totally we can just pass the smaller map, but yeah, let's continue on the other comment :smile: